Core/src/IfcGeometryConverter/SolidModelConverter.h

```diff
@@ -1625,6 +1600,30 @@ namespace OpenInfraPlatform
 				std::copy(firstOperandData->meshsets.begin(), firstOperandData->meshsets.end(), std::back_inserter(itemData->meshsets));
 
 			}
+			
+			/*! \brief Returns an ID of the selected geometric representations. 
+				*
+				* \param[in] ifcOperand				A list of the IDs for geometric representations.
+				* \return							An ID of the selected geometric representations.
+			*/
+			int selectOperand(
+				const typename IfcEntityTypesT::IfcBooleanOperand& ifcOperand
+			) const noexcept(false)
+			{
+				switch (ifcOperand.which()) {
+				case 0:
+					return ifcOperand.get<0>().lock()->getId();
+				case 1:
+					return ifcOperand.get<1>().lock()->getId();
+				case 2:
+					return ifcOperand.get<2>().lock()->getId();
+				case 3:
+					return ifcOperand.get<3>().lock()->getId();
+				default:
+					return 0;
+				}
+			}
+
 
 			void convertIfcBooleanClippingResult(

```

Comment:
IfcBooleanClippingResult is already supported with IfcBooleanResult - it is only a where rule that differentiates these two.

Reply:
Resolved: 0cba750